### ISSUE TITLE
Is there a todo sample?

### ISSUE THREAD

**Issue by aggieben** [fsprojects/FSharp.Data.GraphQL#463]
### Description

Is there a todo sample anywhere?  It's mentioned in the Getting Started section of the main site, but I don't see it in the repo.

![image](https://github.com/fsprojects/FSharp.Data.GraphQL/assets/1321/a9cf6f1f-c1d0-4b53-851e-80e0189f8272)

**Comment by xperiandri** [comment]
Do you mean this file?
https://github.com/fsprojects/FSharp.Data.GraphQL/blob/246d6c037a26e1f91fdbd8c373634f55aa74f4d7/samples/apollo-client/README.md

**Comment by aggieben** [comment]
No, I think I was looking for the server implementation.  What backend would the apollo-client sample be targeted toward?

**Comment by xperiandri** [comment]
Yes, it is for the client type provider demo and tests only

**Comment by aggieben** [comment]
Are there any simple server-side samples, or maybe even other projects that consume FSharp.DataGraphQL?  I'm trying to build something simple in order to learn the lib, and I'm getting lost from the get-go.  Doing the things in the "Getting Started" guide don't work and the Star Wars sample is too much.  

If I need to, I can open a more detailed issue but I thought it was worth asking for other examples.

**Comment by valbers** [comment]
I think you might be interested in taking a look at [the new Giraffe integration](https://github.com/fsprojects/FSharp.Data.GraphQL/blob/dev/src/FSharp.Data.GraphQL.Server.AspNetCore/README.md). I just noticed that the documentation needs to be updated and the part with:
```fsharp
.UseGiraffe
 (HttpHandlers.handleGraphQL<Root>
    applicationLifetime.ApplicationStopping
    (loggerFactory.CreateLogger("FSharp.Data.GraphQL.Server.AspNetCore.HttpHandlers.handleGraphQL"))
 )
```
Should be
```fsharp
.UseGiraffe(HttpHandlers.graphQL<Root>)
 ```

**Comment by valbers** [comment]
Related: #308

**Comment by xperiandri** [comment]
This is the simplest demo
https://github.com/fsprojects/FSharp.Data.GraphQL/blob/releases/2.0.0/samples/relay-modern-starter-kit/server.fsx

Looks like it should be updated too 🙂
I mean paths in `#r` and variables deserialization code
